### ISSUE TITLE
[BUG] Massive Network Frame Drop

### ISSUE THREAD

**Issue by exalented** [mariotaku/moonlight-tv#77]
**Describe the bug**
Client starts experiencing massive frame drop after ~15 minutes of play. This is extremely immersion breaking.

**To Reproduce**
1. Start a session from sunshine server.
2. Use a controller for ~15minutes
3. Client will start dropping anywhere from 50-90% of frames; however, this is not reflected in the Performance States pane. Audio seems unaffected.

**Expected behavior**
0.00% Network Frame Drop hours into a playthrough.

**LG webOS TV:**
 - Model: 65NANO90UNA
 - webOS version: 5.2.1
 - software version: 03.21.80

**Additional context**
This issue is somewhat of a long-standing issue, but has gotten slightly better as more and more gets fixed.
1. Unplugging the controller while dropping frames does nothing.
2. This happens with GeForce Experience and sunshine.
3. There are no logs in sunshine when this happens.
4. This does not happen when using the moonlight-android v9.9.5 client
5. [lag.log](https://github.com/mariotaku/moonlight-tv/files/6706692/lag.log)

**Remedies and speculation**
**This issue is remedied by using the magic remote to simulate a click of the mouse.** After this click the client stops dropping frames and gameplay can resume. This happens continuously, over and over, throughout the session. For this reason I believe this to be some sort of input bug for even resuming an ongoing session breaks game input for me; however, this could easily be related to wine. It seems to happen less often when there's no input by the user. Testing conducted with a wired PS4v1 remote and a wired XBONE controller.

**Comment by mariotaku** [comment]
This is indeed some mysterious bug. I'll share in the channel to see if others are experiencing that.

**Comment by mariotaku** [comment]
Does this still happen? I haven't been able to reproduce yet.

**Comment by exalented** [comment]
Yes this still happens.

**Comment by mariotaku** [comment]
Did you remember when this issue starts?

**Comment by exalented** [comment]
I remember this happening all the way back in 5.2.

**Comment by exalented** [comment]
I am able to reproduce this issue less and less lately. Is anyone else still experiencing this?

**Comment by exalented** [comment]
I was just able to reproduce this and have some more details. Now Network Frame Drop stays at 0.00%, but the client lags and skips frames. Nothing in the details pane seems off. Utilizing the NDL decoder and an XBox One Remote. This happened ~10minutes in without even touching the controller, but once at the beginning on the session.

**Comment by exalented** [comment]
Interesting observation: I was streaming Super Smash Bros Ultimate and I noticed that all of a sudden Yuzu dropped from a very smooth 60fps to 1 fps in the bottom right and frame time showed naan while gamespeed slowed to a near halt. This was observed with Yuzu EA version 1981. Only when I performed the magic remote mitigation did the emulator jump right back up to 60fps; thus, I believe this is some sort of sunshine bug.
